### PR TITLE
Feature/xiaolei/td 1860 node restful

### PR DESCRIPTION
**Pull request by xleili**
pr commit test，skip this till formal commit

### CODE REVIEW

**Review by sangshuduo — CHANGES_REQUESTED** [review]
test program need check result.

File: src/connector/node-rest/src/restResult.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+export class TDengineRestResultSet {
+  constructor(jason) {
+    this.status = ''    //succ
+    this.column_name = {} //head
+    this.column_type = {} //column_meta
+    this.data = {}
+    this.affectRows = null //rows
+    this.code = null
+    this.desc = null
+    this._init(jason)
```
Comment (zitsen) [inline src/connector/node-rest/src/restResult.js]: What is `jason` mean

File: src/connector/node-rest/src/restResult.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        rowStr += cell == null ? 'null' : cell.toString();
+        rowStr += this._fillEmpty(colSize[index] - cell.toString().length) + " | "
+      })
+      console.log(rowStr)
+    })
+
+    return colStr
+  }
+
+  _suggestedMinWidths = {
```
Comment (zitsen) [inline src/connector/node-rest/src/restResult.js]: move it to const variable out of the class.

File: src/connector/node-rest/test/testRestConn.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import {TDRestConnection} from "../TDengineRest";
+
+let conn = new TDRestConnection({host: 'u195'});
+let cursor = conn.cursor();
+
+(async () => {
+  result = await cursor.query("drop database if exists  node_rest");
+  result.toString()
+})()
```
Comment (zitsen) [inline src/connector/node-rest/test/testRestConn.js]: async function may runs after next row, please fix the order.

**Review by zitsen — CHANGES_REQUESTED** [review]
Try to build a better node project from scratch.

**Review by xleili — COMMENTED** [review]
fix test,ci,and async order

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**